### PR TITLE
Adjusted styling

### PR DESCRIPTION
**Pull request by iConic21**
removed redundant scroll bars and are only shown when there is coming to scroll.
added a hint text in the right menu list on how to invite new members
before
after

### CODE REVIEW

File: src/index.js [hunk 1]
```diff
@@ -223,6 +223,8 @@ class View extends React.Component {
     } = this.state
     const { createRoom, createConvo, removeUserFromRoom } = this.actions
 
+    console.log(messages);
```
Comment (mdpye) [inline src/index.js]: This probably doesn't want to make it on to master.

Comment (iConic21) [reply src/index.js]: oops missed that one, good pick up. Its removed now.

Comment (iConic21) [reply src/index.js]: @mdpye can you re-review please?

**Comment by mdpye** [comment]
Thanks @iConic21

**Pull request merged**